### DISCUSSION TITLE
Feasibility of using flutter_rust_bridge for polars bindings

### DISCUSSION THREAD

**Discussion by JordyScript** [fzyzcjy/flutter_rust_bridge#2273] [General]
I'm wondering how practical it would be to generate dart bindings for polars (the Rust dataframe library) using flutter_rust_bridge. Has anyone tried this? How did it go? Are necessary features  missing from flutter_rust_bridge? Would it be better/easier to write bindings by hand for a library of this size? 

Dart and Flutter really need a package that can work with tabular data and bindings to polars seem the most pragmatic solution for that to happen.

**Comment by fzyzcjy** [comment]
@Desdaemon has done a great job to (experimentally) export polars: https://github.com/Desdaemon/polars_dart

(Btw I also use polars frequently and it is quite useful!)

**Reply by Desdaemon** [reply]
Yeah, and part of polars-dart was meant to dogfood some of the more obscure features of flutter_rust_bridge. Though most of these days I work with Python codebases so I don't work on this much more, feel free to play around to see if it at least compiles today.

**Comment by Desdaemon** [comment]
Also my two cents from writing polars-dart, the entire library is massive (including default features alone produces a 30MB-ish dylib) so you might get more mileage out of actually using polars in your FRB crate and only installing features that you use.
